platform/MCU/nRF52840/inc/dct.h

```diff
@@ -77,7 +77,8 @@ typedef struct __attribute__((packed)) application_dct {
     uint8_t device_private_key[1216];   // sufficient for 2048 bits
     uint8_t device_public_key[384];     // sufficient for 2048 bits
     static_ip_config_t  ip_config;
-    uint8_t unused[96];
+    uint8_t unused[95];
+    uint8_t ota_update_flag;            // should be 0xAA55 to trigger an update from data stored in the update region
```

Comment:
The comment is outdated: `should be 0xAA55` -> `should be 0xA5`. Same [here](https://github.com/particle-iot/device-os/pull/1788/files#diff-4891cc98f0686787be0bf0e4bb6a11e0R78)

Reply:
Fixed.